backends-clickhouse/src/test/scala/io/glutenproject/execution/GlutenClickHouseTPCHColumnarShuffleParquetAQESuite.scala

```diff
@@ -254,4 +256,16 @@ class GlutenClickHouseTPCHColumnarShuffleParquetAQESuite
         assert(adaptiveSparkPlanExec(1) == adaptiveSparkPlanExec(2))
     }
   }
+
+  test("Test 'spark.gluten.enabled' false") {
+    withSQLConf(("spark.gluten.enabled", "false")) {
```

Comment:
is it needed to set `spark.gluten.enabled` to false and true to run test?

Reply:
not needed, the other test cases run with spark.gluten.enabled true,